src/dmd/backend/disasm86.d

```diff
@@ -0,0 +1,3758 @@
+                }
+                break;
+            case 0xDD:
+                if (mod != 3)
+                {   p1 = opDDa[reg];
+                    p2 = getEA(0, c);
+                    if (reg == 1)               // if FISTTP m64int
+                        mfp = "long64";
+                }
+                else if (reg <= 5)
+                {   p1 = opDDb[reg];
+                    if (modrgrm & 7)
+                    {
+                        strcpy(sti.ptr, STI);
+                        sti[3] = (modrgrm & 7) + '0';
+                        p2 = sti.ptr;
+                    }
+                    else
+                        p2 = ST;
+                }
+                break;
+            case 0xDF:
+                p1 = op7[reg];
+                if (reg == 1)
+                    mfp = "short";
+                else if (reg <= 3)
+                    mfp = "long";
+                else if (reg == 5 || reg == 7)
+                    mfp = "long64";
+                if ((modrgrm & 0xC0) == 0xC0)
+                {   p1 = (reg <= 6) ? op7b[reg] : reserved;
+                    if (reg == 4)
+                        p2 = "AX";
+                    else
+                    {   if (reg == 5)
+                            mfp = "";
+                        if ((modrgrm & 7) != 1)
+                        {
+                            strcpy(sti.ptr, STI);
+                            sti[3] = (modrgrm & 7) + '0';
+                            p2 = sti.ptr;
+                        }
+                    }
+                }
+                else
+                    p2 = getEA(0, c);
+                break;
+            default:
+                break;
+        }
+    }
+L2:
+    void puts(const(char)* s)
+    {
+        while (*s)
+        {
+            put(*s);
+            ++s;
+        }
+    }
+
+    puts(p0);
+    put('\t');
+    if (waitflag)
+        puts(p1 + 2);
+    else
+        puts(p1);
+    if (*p2)
+    {
+        put('\t');
+        if (*mfp)
+        {
+            puts(mfp);
+            puts(" ptr ");
+        }
+        puts(p2);
+        if (*p3)
+        {
+            put(',');
+            puts(p3);
+        }
+    }
+}
+
+void puts(const(char)* s)
+{
+    while (*s)
+    {
+        put(*s);
+        ++s;
+    }
+}
+
+
+/*************************
+ * Disassemble the instruction at `c`
+ * Params:
+ *      c = index into code[]
+ */
+
+void disassemble(uint c)
+{
+    //printf("disassemble(c = %d, siz = %d)\n", c, siz);
+    uint prefixsize = 0;
+    uint sz;
+    do
+    {
+        sz = prefixbyte(c);
+        c += sz;
+        prefixsize += sz;
+    } while (sz);
+    assert(siz > prefixsize);
+    siz -= prefixsize;
+
+    put('\t');
+
+    uint opcode,reg = 0;
+    ubyte rex;
+    int i,o3;
+    const(char)* p1, p2, p3, p4;
+    char[80] p0;
+    char[5] p1buf;
+    const(char)* sep;
+    const(char)* s2;
+    const(char)* s3;
+    char[BUFMAX] buf = void;
+    enum MOV = "mov";
```

Comment:
```suggestion
    static immutable MOV = "mov";
```
Avoid using `enum` for arrays, appending to it might not generate a new compile-time string and instead calls to append it at runtime.